lib/Immediate/CMakeLists.txt

```diff
@@ -12,9 +12,9 @@ target_link_libraries(swiftImmediate PRIVATE
   swiftIRGen
   swiftSILGen
   swiftSILOptimizer)
-if(LibEdit_FOUND)
+if(LibEdit_HAS_UNICODE)
```

Comment:
Can you make this `LibEdit_FOUND AND LibEdit_HAS_UNICODE`?

Reply:
Sure.